readme.md

```diff
@@ -1,6 +1,6 @@
 # Stopwatch task
 Replace `<your_account>` with your Github username and copy the links to Pull Request description:
-- [DEMO LINK](https://<your_account>.github.io/layout_stop-watch/)
+- [DEMO LINK](https://lefteris-l.github.io/layout_stop-watch/)
```

Comment:
The demo link provided doesn't seem to work. Maybe you didn't deploy yet?

npm run deploy

should make it work

Reply:
It should work now